Dockerfile

```diff
@@ -95,7 +96,7 @@ RUN sed -i 's/^# en_US.UTF-8 UTF-8$/en_US.UTF-8 UTF-8/g' /etc/locale.gen \
 # Install Hadoop
 RUN cd /tmp && \
     wget -q https://archive.cloudera.com/cdh5/cdh/5/hadoop-${HADOOP_VERSION}-cdh5.11.0.tar.gz && \
-    tar xzf hadoop-${HADOOP_VERSION}-cdh5.11.0.tar.gz --strip-components 1 -C $HADOOP_HOME && \
+    tar xzf hadoop-${HADOOP_VERSION}-cdh5.11.0.tar.gz --absolute-names --strip-components 1 -C $HADOOP_HOME && \
```

Comment:
What's this flag change for?

It sounds someone counter-intuative when combined with -C -- absolute-names means any absolute paths in the tar will be extracted to there, but with strip-components, and -C mean we want to extract into to a specific subdir...?

Reply:
I ran into [this](https://github.com/docker/hub-feedback/issues/727) issue when building this docker image. It seems to be a sporadic issue related to using tar in docker, I haven't looked deeper than someone mentioned that `--absolute-names` helps, which I tried and it builds now. 

There are no absolute paths in the hadoop tar so as you say this actually has no effect, but it must change something internal to tar since my build fails without it:

```
Step 21/29 : RUN cd /tmp &&     wget -q https://archive.cloudera.com/cdh5/cdh/5/hadoop-${HADOOP_VERSION}-cdh5.11.0.tar.gz &&     tar xzf hadoop-${HADOOP_VERSION}-cdh5.11.0.tar.gz --strip-components 1 -C $HADOOP_HOME &&     rm hadoop-${HADOOP_VERSION}-cdh5.11.0.tar.gz
 ---> Running in 406a118eff9e
tar: share/hadoop/mapreduce1: Directory renamed before its status could be extracted
tar: Exiting with failure status due to previous errors
```

It may not be a problem for the automated build of this repository so I can remove it. Thoughts?

Reply:
Lets keep it, but just add a comment in to the docker file why we have it.

(Cos of dockerfile semantics it should be like:

```
# --absolute-names is cos of X
RUN cd /tmp && \
...
```
)

Reply:
Done.